tests/parent.spec.js

```diff
@@ -48,8 +50,7 @@ describe('Parent', () => {
 			expect(Parent).toBeDefined();
 
 			let parent = new Parent();
-			expect(parent.startPollingTabs).toBeDefined();
-			expect(parent.watchStatus).toBeDefined();
+			expect(parent.onChildUnload).toBeDefined();
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 6 spaces but found 3 tabs.](https://app.codacy.com/gh/wingify/across-tabs/pullRequest?prid=7659180)
```suggestion
      expect(parent.onChildUnload).toBeDefined();
```